docs/providers/Providers Guides/google_fit.mdx

```diff
@@ -0,0 +1,17 @@
+# Google Fit
+
+## Data accuracy
+
+Activity data fetched from Google Fit might be slightly different than the readings in the Google Fit app. Calories data is very accurate - the biggest difference we've seen is around 3-4kcal over a total of ~2k. Step count data can be more off - the biggest difference we observed is around 250-300 steps over a total count of ~10k.
+
+This can happen for a variety of reasons:
+
+* Differences in recent data can be due to syncing delays. From Google's docs:
+    > This temporary difference in numbers could be due to syncing delays. Our backend has the most recent version of data and your app will show the same values once it's synced with the backend.
+* Bug fixes on Google's side:
+    > Sometimes the step calculation code for the Google Fit app is updated with bug fixes before we are able to release the fixes to developers (which requires a Google Play Services release). We are also working on making it possible for developers to access fixes at the same time.
+* The device could have values which are not yet synced to the server.
+
+For more information on this, you can check out the [FAQ page](https://developers.google.com/fit/faq#how_do_i_get_the_same_values_step_count_calories_distance_etc_as_the_google_fit_app) in the Google Fit API documentation.
+
+We are working to make the data we fetch as close as possible to the in-app readings.
```

Comment:
This is a great explanation. 👍 Well written.

Reply:
Thanks @RuiAAPeres 😄